packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -12,6 +12,7 @@ data class GutenbergProps @JvmOverloads constructor(
     val enableAudioBlock: Boolean,
     val localeSlug: String,
     val postType: String,
+    val featuredMedia: Int,
```

Comment:
Kinda a nit but, wdyt about just using `featuredImageId` pretty much everywhere, except the Post object itself where the property is named `featuredMedia`? Both the apps and Gutenberg-desktop refer to "featured image" on the UI and code. I feel like using the `Media` part in places just adds some confusion. Thoughts?

Reply:
I'd had this on my mind a little too, I used `featuredMedia` as it was a bit more inline with `getEditedPostAttribute('featured_media')`, but agree that using `featuredImageId` seems clearer overall. I've updated this PR and will update the Android PR too when the tests have run.

Reply:
Should also add that I'm not sure if WordPress would accept a _video_ as featured on a post, in which case "featured media" is a more appropriate name. Gutenberg-desktop doesn't seem to offer videos when selecting a featured media though.